File: data/recomposed_projects-manual-old/deepseek-coder-33b-instruct/body/commons-validator/evosuite-test/src/test/org/apache/commons/validator/ValidatorResults_ESTest.py
from __future__ import annotations
import time
import re
import os
import unittest
import pytest
import io
import unittest
from src.main.org.apache.commons.validator.Field import *
from src.main.org.apache.commons.validator.ValidatorResult import *
from src.main.org.apache.commons.validator.ValidatorResults import *

# from src.test.org.apache.commons.validator.ValidatorResults_ESTest_scaffolding import *
# from src.main.org.evosuite.runtime.EvoAssertions import *
# from src.main.org.evosuite.runtime.EvoRunnerParameters import *


class ValidatorResults_ESTest(unittest.TestCase):

    def test17(self) -> None:

        validatorResults0 = ValidatorResults()
        set0 = validatorResults0.getPropertyNames()
        self.assertEqual(0, len(set0))

    def test16(self) -> None:

        validatorResults0 = ValidatorResults()
        self.assertTrue(validatorResults0.isEmpty())

        field0 = Field()
        validatorResults0.add0(field0, "[]", True)
        boolean0 = validatorResults0.isEmpty()
        self.assertFalse(boolean0)

    def test15(self) -> None:
        validatorResults0 = ValidatorResults()
        validatorResults0.merge(validatorResults0)
        self.assertTrue(validatorResults0.isEmpty())

    def test14(self) -> None:
        validatorResults0 = ValidatorResults()
        validatorResults0.clear()
        self.assertTrue(validatorResults0.isEmpty())

    def test13(self) -> None:

        validatorResults0 = ValidatorResults()
        boolean0 = validatorResults0.isEmpty()
        self.assertTrue(boolean0)

    def test12(self) -> None:

        validatorResults0 = ValidatorResults()
        field0 = Field()
        validatorResults0.add0(field0, "[]", True)
        map0 = validatorResults0.getResultValueMap()
        self.assertTrue(not map0)

    def test11(self) -> None:

        validatorResults0 = ValidatorResults()
        field0 = Field()
        object0 = object()
        validatorResults0.add1(field0, "[]", False, object0)
        map0 = validatorResults0.getResultValueMap()
        self.assertFalse(map0)

    def test10(self) -> None:
        validatorResults0 = ValidatorResults()
        validatorResult0 = validatorResults0.getValidatorResult(None)
        self.assertIsNone(validatorResult0)

    def test09(self) -> None:

        validatorResults0 = ValidatorResults()
        field0 = Field()
        object0 = object()
        validatorResults0.add1(field0, "", False, object0)
        validatorResults0.add1(field0, "~+^*", False, object0)
        self.assertIsNone(field0.getProperty())

    def test08(self) -> None:

        validatorResults0 = ValidatorResults()
        try:
            validatorResults0.add0(None, "L*", True)
            self.fail("Expecting exception: RuntimeError")

        except Exception as e:
            self.verifyException("org.apache.commons.validator.ValidatorResults", e)

    def test07(self) -> None:

        validatorResults0 = ValidatorResults()

        try:
            validatorResults0.add1(None, "", True, "")
            self.fail("Expecting exception: RuntimeError")

        except Exception as e:
            self.assertIsInstance(e, RuntimeError)
            self.assertIsNone(e.getMessage())

    def test06(self) -> None:

        validatorResults0 = ValidatorResults()
        validatorResults0.hResults = None

        with self.assertRaises(RuntimeError):
            validatorResults0.clear()
            self.fail("Expecting exception: RuntimeError")

        verifyException("org.apache.commons.validator.ValidatorResults", RuntimeError)

    def test05(self) -> None:

        validatorResults0 = ValidatorResults()
        validatorResults0.hResults = None

        with self.assertRaises(RuntimeError):
            validatorResults0.getPropertyNames()

    def test04(self) -> None:

        validatorResults0 = ValidatorResults()
        validatorResults0.hResults = None

        try:
            validatorResults0.getResultValueMap()
            self.fail("Expecting exception: RuntimeError")

        except Exception as e:
            self.assertEqual(str(type(e)), "<class 'RuntimeError'>")

    def test03(self) -> None:

        validatorResults0 = ValidatorResults()
        validatorResults0._hResults = None

        with self.assertRaises(RuntimeError):
            validatorResults0.getValidatorResult("[]")

    def test02(self) -> None:

        validatorResults0 = ValidatorResults()
        validatorResults0._hResults = None

        with pytest.raises(RuntimeError):
            validatorResults0.isEmpty()

    def test01(self) -> None:

        validatorResults0 = ValidatorResults()

        try:
            validatorResults0.merge(None)
            self.fail("Expecting exception: RuntimeError")

        except Exception as e:
            self.assertEqual(str(e), "Expecting exception: RuntimeError")

    def test00(self) -> None:

        validatorResults0 = ValidatorResults()
        field0 = Field()
        validatorResults0.add0(
            field0, "org.apache.commons.validator.ValidatorResult$ResultStatus", False
        )
        validatorResult0 = validatorResults0.getValidatorResult(None)
        self.assertIsNotNone(validatorResult0)

File: java_projects/cleaned_final_projects_evosuite/commons-validator/src/test/java/org/apache/commons/validator/ValidatorResults_ESTest.java


/*
  Licensed to the Apache Software Foundation (ASF) under one or more
  contributor license agreements.  See the NOTICE file distributed with
  this work for additional information regarding copyright ownership.
  The ASF licenses this file to You under the Apache License, Version 2.0
  (the "License"); you may not use this file except in compliance with
  the License.  You may obtain a copy of the License at

      http://www.apache.org/licenses/LICENSE-2.0

  Unless required by applicable law or agreed to in writing, software
  distributed under the License is distributed on an "AS IS" BASIS,
  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  See the License for the specific language governing permissions and
  limitations under the License.
 */



package org.apache.commons.validator;

import org.junit.Test;
import static org.junit.Assert.*;
import static org.evosuite.runtime.EvoAssertions.*;
import java.util.Map;
import java.util.Set;
import org.apache.commons.validator.Field;
import org.apache.commons.validator.ValidatorResult;
import org.apache.commons.validator.ValidatorResults;
import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.runner.RunWith;

@RunWith(EvoRunner.class) @EvoRunnerParameters(mockJVMNonDeterminism = true, useVFS = true, useVNET = true, resetStaticState = true) 
public class ValidatorResults_ESTest extends ValidatorResults_ESTest_scaffolding {

  @Test(timeout = 4000)
  public void test00()  throws Throwable  {
      ValidatorResults validatorResults0 = new ValidatorResults();
      Field field0 = new Field();
      validatorResults0.add0(field0, "org.apache.commons.validator.ValidatorResult$ResultStatus", false);
      ValidatorResult validatorResult0 = validatorResults0.getValidatorResult((String) null);
      assertNotNull(validatorResult0);
  }

  @Test(timeout = 4000)
  public void test01()  throws Throwable  {
      ValidatorResults validatorResults0 = new ValidatorResults();
      // Undeclared exception!
      try { 
        validatorResults0.merge((ValidatorResults) null);
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         verifyException("org.apache.commons.validator.ValidatorResults", e);
      }
  }

  @Test(timeout = 4000)
  public void test02()  throws Throwable  {
      ValidatorResults validatorResults0 = new ValidatorResults();
      validatorResults0.hResults = null;
      // Undeclared exception!
      try { 
        validatorResults0.isEmpty();
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
      }
  }

  @Test(timeout = 4000)
  public void test03()  throws Throwable  {
      ValidatorResults validatorResults0 = new ValidatorResults();
      validatorResults0.hResults = null;
      // Undeclared exception!
      try { 
        validatorResults0.getValidatorResult("[]");
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         verifyException("org.apache.commons.validator.ValidatorResults", e);
      }
  }

  @Test(timeout = 4000)
  public void test04()  throws Throwable  {
      ValidatorResults validatorResults0 = new ValidatorResults();
      validatorResults0.hResults = null;
      // Undeclared exception!
      try { 
        validatorResults0.getResultValueMap();
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         verifyException("org.apache.commons.validator.ValidatorResults", e);
      }
  }

  @Test(timeout = 4000)
  public void test05()  throws Throwable  {
      ValidatorResults validatorResults0 = new ValidatorResults();
      validatorResults0.hResults = null;
      // Undeclared exception!
      try { 
        validatorResults0.getPropertyNames();
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         verifyException("org.apache.commons.validator.ValidatorResults", e);
      }
  }

  @Test(timeout = 4000)
  public void test06()  throws Throwable  {
      ValidatorResults validatorResults0 = new ValidatorResults();
      validatorResults0.hResults = null;
      // Undeclared exception!
      try { 
        validatorResults0.clear();
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         verifyException("org.apache.commons.validator.ValidatorResults", e);
      }
  }

  @Test(timeout = 4000)
  public void test07()  throws Throwable  {
      ValidatorResults validatorResults0 = new ValidatorResults();
      // Undeclared exception!
      try { 
        validatorResults0.add1((Field) null, "", true, "");
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         verifyException("org.apache.commons.validator.ValidatorResults", e);
      }
  }

  @Test(timeout = 4000)
  public void test08()  throws Throwable  {
      ValidatorResults validatorResults0 = new ValidatorResults();
      // Undeclared exception!
      try { 
        validatorResults0.add0((Field) null, "L*", true);
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         verifyException("org.apache.commons.validator.ValidatorResults", e);
      }
  }

  @Test(timeout = 4000)
  public void test09()  throws Throwable  {
      ValidatorResults validatorResults0 = new ValidatorResults();
      Field field0 = new Field();
      Object object0 = new Object();
      validatorResults0.add1(field0, "", false, object0);
      validatorResults0.add1(field0, "~+^*", false, object0);
      assertNull(field0.getProperty());
  }

  @Test(timeout = 4000)
  public void test10()  throws Throwable  {
      ValidatorResults validatorResults0 = new ValidatorResults();
      ValidatorResult validatorResult0 = validatorResults0.getValidatorResult((String) null);
      assertNull(validatorResult0);
  }

  @Test(timeout = 4000)
  public void test11()  throws Throwable  {
      ValidatorResults validatorResults0 = new ValidatorResults();
      Field field0 = new Field();
      Object object0 = new Object();
      validatorResults0.add1(field0, "[]", false, object0);
      Map<String, Object> map0 = validatorResults0.getResultValueMap();
      assertFalse(map0.isEmpty());
  }

  @Test(timeout = 4000)
  public void test12()  throws Throwable  {
      ValidatorResults validatorResults0 = new ValidatorResults();
      Field field0 = new Field();
      validatorResults0.add0(field0, "[]", true);
      Map<String, Object> map0 = validatorResults0.getResultValueMap();
      assertTrue(map0.isEmpty());
  }

  @Test(timeout = 4000)
  public void test13()  throws Throwable  {
      ValidatorResults validatorResults0 = new ValidatorResults();
      boolean boolean0 = validatorResults0.isEmpty();
      assertTrue(boolean0);
  }

  @Test(timeout = 4000)
  public void test14()  throws Throwable  {
      ValidatorResults validatorResults0 = new ValidatorResults();
      validatorResults0.clear();
      assertTrue(validatorResults0.isEmpty());
  }

  @Test(timeout = 4000)
  public void test15()  throws Throwable  {
      ValidatorResults validatorResults0 = new ValidatorResults();
      validatorResults0.merge(validatorResults0);
      assertTrue(validatorResults0.isEmpty());
  }

  @Test(timeout = 4000)
  public void test16()  throws Throwable  {
      ValidatorResults validatorResults0 = new ValidatorResults();
      assertTrue(validatorResults0.isEmpty());
      
      Field field0 = new Field();
      validatorResults0.add0(field0, "[]", true);
      boolean boolean0 = validatorResults0.isEmpty();
      assertFalse(boolean0);
  }

  @Test(timeout = 4000)
  public void test17()  throws Throwable  {
      ValidatorResults validatorResults0 = new ValidatorResults();
      Set<String> set0 = validatorResults0.getPropertyNames();
      assertEquals(0, set0.size());
  }
}
